NOTICE

```diff
@@ -1,4 +1,4 @@
 Datadog .NET Tracer for APM (dd-trace-dotnet)
-Copyright 2018 Datadog, Inc.
+Copyright (C) Datadog, Inc. 2017
```

Comment:
I don't know about the other notices, but the template in the Apache license text (of which this `NOTICE` is a part of)  is `Copyright [yyyy] [name of copyright owner]`.

```suggestion
Copyright 2017 Datadog, Inc.
```

Reply:
I'll update all instances to use this version requested by the Apache license, stay tuned

Reply:
Updated in d3b105596b374c0379f917a35e17d0ec17902426